introduction.md

```diff
@@ -2,3 +2,7 @@ test Oh seriously it should be  long enough
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?